### PR TITLE
fix issue #196

### PR DESCRIPTION
**Pull request by Roelandsmith**
#196 opened yesterday by Roelandsmith
Change text in dialoge box 'unsaved changes' enhancement

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "CANCEL": "Cancel",
+  "CANCEL": "I'll save them first",
```
Comment (paulleaphy) [inline src/assets/i18n/en.json:2]: Translation keys can be used in multiple locations in the application. This is an example. So when you change this to "I'll save them first", you are changing that in all places. I doubt that is what you want. You probably want to change it just for the editor-change confirmation dialog. So what you want is to create a new translation key and value in both language files and reference the new key inside the dialog.

File: src/assets/i18n/nl.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "CANCEL": "Annuleer",
+  "CANCEL": "Ik ga ze eerst opslaan",
```
Comment (paulleaphy) [inline src/assets/i18n/nl.json:2]: See my comment in the en.json file.

File: src/assets/i18n/en.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
   "REFRESH": "Refresh",
   "CLOSE": "Close",
-  "OK": "Ok",
+  "OK": "That's okay",
```
Comment (paulleaphy) [inline src/assets/i18n/en.json:5]: The same applies here, this change will change all instances of "Ok" to "That's okay", which might look kind of weird in some dialogs.

File: src/assets/i18n/nl.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
   "REFRESH": "Ververs",
   "CLOSE": "Sluiten",
-  "OK": "Ik begrijp het",
+  "OK": "Dat is oké",
```
Comment (paulleaphy) [inline src/assets/i18n/nl.json:5]: See my comment in the en.json file.

File: src/assets/i18n/nl.json [hunk 5]
```diff
@@ -64,6 +64,6 @@
   "CODE": "Code",
   "BLOCKS": "Blokken",
   "DIRECT_TO_CODE_EDITOR": "Direct naar Code-scherm",
-  "CONFIRMEDITORCHANGE_TITLE": "Onopgeslagen wijzigingen",
-  "EDITORCHANGEINSTRUCTIONS": "Je hebt onopgeslagen wijzigingen. Weet je zeker dat je terug wilt naar de blokken editor? Je wijzigingen zullen verloren raken!"
+  "CONFIRMEDITORCHANGE_TITLE": "Wijzigingen opslaan?",
+  "EDITORCHANGEINSTRUCTIONS": "Je hebt onopgeslagen wijzigingen. \rAls je nu naar het Blokken-scherm gaat, ben je die kwijt."
```
Comment (paulleaphy) [inline src/assets/i18n/nl.json:68]: There is a carriage return character (\r) in here that is not in the en.json. Is that intentional?

**Review by paulleaphy — CHANGES_REQUESTED** [review]

File: src/assets/i18n/nl.json (hunk 2 above)
Comment (Roelandsmith) [reply src/assets/i18n/nl.json:2]: Understood!
The short cut is not a neat cut:)
Are you willing to fix that?
I think it's quicker...

File: src/assets/i18n/en.json (hunk 3 above)
Comment (Roelandsmith) [reply src/assets/i18n/en.json:5]: Idem!

File: src/assets/i18n/nl.json (hunk 5 above)
Comment (Roelandsmith) [reply src/assets/i18n/nl.json:68]: No! I was experementing with a way to start on a new line...but it didn't work out.
And apparently i didn't remove one of the tries...

**Comment by paulleaphy** [comment]
As discussed, I will make the needed changes in a separate PR

**Pull request closed without merging**